setup.py

```diff
@@ -44,7 +44,11 @@
         'mozilla_django_oidc',
     ],
     include_package_data=True,
-    install_requires=['Django>1.7', 'python-jose', 'requests'],
+    install_requires=[
+        'Django>1.7',
```

Comment:
I suspect this should be `Django>=1.8`. Does this library support Django 1.7? If so, there's no testing for it.